packages/server/__snapshots__/7_record_spec.js

```diff
@@ -1531,10 +1533,254 @@ https://on.cypress.io/dashboard/organizations/org-id-1234/billing
 
 `
 
-exports['e2e record api interaction warnings create run warnings grace period - parallel feature warns when using parallel feature 1'] = `
-Parallelization is not included under your free plan.
+exports['e2e record passing passes 2'] = [
```

Comment:
this PR inadvertently moved around the snapshot code... but I manually verified that this data object diff matches the previous one... so we're good